RecoTracker/FinalTrackSelectors/src/TfGraphDefWrapper.cc

```diff
@@ -1,6 +1,12 @@
 #include "RecoTracker/FinalTrackSelectors/interface/TfGraphDefWrapper.h"
 
-TfGraphDefWrapper::TfGraphDefWrapper(tensorflow::Session* session) : session_(session) {}
+TfGraphDefWrapper::TfGraphDefWrapper(tensorflow::Session* session, tensorflow::GraphDef* graph)
+    : session_(session), graph_(graph) {}
 const tensorflow::Session* TfGraphDefWrapper::getSession() const {
   return const_cast<const tensorflow::Session*>(session_);
```

Comment:
By the way, this explicit `const_cast` is unnecessary (a non-const pointer casts implicitly to a const pointer). It doesn't hurt for the compiler, only for the eye that views all `const_cast`s as potentially harmful :)

Reply:
yes, this might be removed, thx for pointing out!